packages/jsii-pacmak/lib/targets/dotnet/dotnetgenerator.ts

```diff
@@ -833,7 +832,7 @@ export class DotNetGenerator extends Generator {
       );
       if (implementedInBase || datatype || proxy) {
         // Override if the property is in a datatype or proxy class or declared in a parent class
-        isOverrideKeyWord = 'override ';
+        isOverrideKeyWord = prop.static ? 'new ' : 'override ';
```

Comment:
Oh god, it took me a moment to understand what the `new` keyword is doing here. Can we add a link to Microsoft documentation? https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/new-modifier
Added as a suggestion to the doc line.